empiric-package/empiric/core/entry.py

```diff
@@ -1,21 +1,22 @@
 from collections import namedtuple
+from typing import Any, List
 
 from empiric.core.utils import str_to_felt
 
 Entry = namedtuple("Entry", ["key", "value", "timestamp", "source", "publisher"])
 
 
-def serialize_entry(entry):
+def serialize_entry(entry) -> List[Any]:
     return [entry.key, entry.value, entry.timestamp, entry.source, entry.publisher]
 
 
-def serialize_entries(entries):
+def serialize_entries(entries) -> List[Any]:
     expanded = [serialize_entry(entry) for entry in entries]
     flattened = [x for entry in expanded for x in entry]
     return [len(entries)] + flattened
```

Comment:
I was curious why this returned a flattened list of entries in addition to an int, it's an unusual makeup for a list?

Reply:
I believe this is due to what parameters the StarkNet code expects. To represent an array you pass in the length of the array and then all the objects individually afterwards. (This is exactly what happens here.) I assume that's why the format is like that.

Reply:
Makes sense